pkg/cmd/kafka/list/list.go

```diff
@@ -38,46 +37,38 @@ type kafkaRow struct {
 }
 
 type options struct {
-	outputFormat string
-	page         int
-	limit        int
-	search       string
-
-	IO             *iostreams.IOStreams
-	Connection     factory.ConnectionFunc
-	Logger         logging.Logger
-	localizer      localize.Localizer
-	Context        context.Context
-	ServiceContext servicecontext.IContext
+	outputFormat            string
+	page                    int
+	limit                   int
+	search                  string
+	accessToken             string
+	clusterManagementApiUrl string
+
+	f *factory.Factory
 }
 
 // NewListCommand creates a new command for listing kafkas.
 func NewListCommand(f *factory.Factory) *cobra.Command {
 	opts := &options{
-		page:           0,
-		limit:          100,
-		search:         "",
-		Connection:     f.Connection,
-		Logger:         f.Logger,
-		IO:             f.IOStreams,
-		localizer:      f.Localizer,
-		Context:        f.Context,
-		ServiceContext: f.ServiceContext,
+		page:   0,
```

Comment:
why are we setting these here? the best place comes from the default values in the flag?

Reply:
Weird not sure why I added that 🤔